controllers/setup.go

```diff
@@ -139,13 +139,22 @@ func setupManager(ctx context.Context, cancel context.CancelFunc, mgr controller
 		return fmt.Errorf("error adding service controller: %w", err)
 	}
 
-	vmController, err := CreateVmController(mgr)
+	hasKubeVirtCRDs, err := common.CheckIfClusterHasKubeVirtCRDs(ctx, mgr.GetAPIReader())
 	if err != nil {
-		return fmt.Errorf("failed to create vm controller: %w", err)
-	}
+		return fmt.Errorf("[vm controller] failed to check if cluster has KubeVirt CRDs: %w", err)
+	} else if hasKubeVirtCRDs {
```

Comment:
This can be a plain `if`, no need for `else`.

Reply:
updated